wallet/src/Dscp/Wallet/Backend.hs

```diff
@@ -96,6 +98,27 @@ sendTx wc sendEvent eSecretKey mPassPhrase (toList -> outs) = do
     sendStateUpdateEvent sendEvent
     return (twTx txWitnessed)
 
+txFee ::
+       HasCoreConfig
+    => SecretKey
+    -> NonEmpty TxOut
+    -> Coin
+txFee dummySK (toList -> outs) = unFees $ calcFeeG feeCoefficients gTx
+  where
+    publicKey = toPublic dummySK
+    address = mkAddr publicKey
+    nonce = 0
+
+    gTx = GMoneyTxWitnessed . fixFees feeCoefficients $ \fees ->
+        let inAcc   = TxInAcc { tiaNonce = nonce, tiaAddr = address }
+            inValue = Coin (sum $ map (unCoin . txOutValue) outs) `unsafeAddCoin` unFees fees
+            tx      = Tx { txInAcc = inAcc, txInValue = inValue, txOuts = outs }
+
+            signature   = sign dummySK (toTxId tx, publicKey, ())
+            witness     = TxWitness   { txwSig = signature, txwPk = publicKey }
+            txWitnessed = TxWitnessed { twTx   = tx, twWitness = witness }
+        in txWitnessed
+
```

Comment:
I think it should be in `Dscp.Core.Fees` module

Reply:
And actually this should not require a secret key. All witnesses have the same size, we can just have hardcoded "sample" witness for this purpose.
I disagree with @Martoon-00's "moon phase" argument; I claim that fee size will not depend on moon phase (or the identity of a sender) in the nearest future, I see no possible reason for it to depend on it and I doubt that we should have suboptimal code "just in case". If we'll find it necessary, we'll introduce such a dependency separately anyway.

Reply:
What is the preferred way of _hardcoding witness_? Coercing some hardcoded stuff to `Signature` and `PublicKey`?